GoogleDataTransport/GDTCORLibrary/GDTCORConsoleLogger.m

```diff
@@ -16,40 +16,46 @@
 
 #import "GoogleDataTransport/GDTCORLibrary/Public/GoogleDataTransport/GDTCORConsoleLogger.h"
 
-volatile NSInteger GDTCORConsoleLoggerLoggingLevel = GDTCORLoggingLevelErrors;
+#import <GoogleUtilities/GULLogger.h>
+
+volatile NSInteger GDTCORConsoleLoggerLoggingLevel = GDTCORLoggingLevelWarnings;
 
 /** The console logger prefix. */
 static NSString *kGDTCORConsoleLogger = @"[GoogleDataTransport]";
 
 NSString *GDTCORMessageCodeEnumToString(GDTCORMessageCode code) {
-  return [[NSString alloc] initWithFormat:@"I-GDTCOR%06ld", (long)code];
+  return [[NSString alloc] initWithFormat:@"I-GDT%06ld", (long)code];
 }
 
 void GDTCORLog(GDTCORMessageCode code, GDTCORLoggingLevel logLevel, NSString *format, ...) {
-// Don't log anything in not debug builds.
-#if !NDEBUG
```

Comment:
I imagine the  `!NDEBUG` check was added so that production builds avoid this logic and associated logging.

Does this code get executed enough that we need to worry about the performance impact?

Reply:
Yeah probably worth putting back. Debug logs are pretty noisy.